src/components/About/Team/index.js

```diff
@@ -133,18 +131,81 @@ const Description = styled.div`
   padding: 1rem;
 `;
 
+const contributorsWebLink =
+  'https://api.github.com/repos/fossasia/susi.ai/contributors';
+const contributorsServerLink =
+  'https://api.github.com/repos/fossasia/susi_server/contributors';
 export default class Team extends Component {
   static propTypes = {
     history: PropTypes.object,
     location: PropTypes.object,
   };
 
-  componentDidMount() {
+  state = {
+    contributors: [],
```

Comment:
Also, can you use hooks here and convert the component to a functional one?

Reply:
Sir , I have also opened an issue #3431  to convert the class based components to  functional  components